src/Uno.UI.Runtime.Skia.Gtk/GTK/UnoDrawingArea.cs

```diff
@@ -9,14 +9,31 @@ internal class UnoDrawingArea : Gtk.DrawingArea
 		private SKBitmap bitmap;
 		private int renderCount;
 		private int InvalidateRenderCount;
+		private double _dpi;
 
 		public UnoDrawingArea()
 		{
 			WUX.Window.Current.InvalidateRender
 				+= () =>
 				{
-					QueueDrawArea(0, 0, 10000, 1000);
+					Invalidate();
 				};
+
+			//Window.Screen.MonitorsChanged += Screen_MonitorsChanged;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)